experimental/packages/opentelemetry-instrumentation-http/test/functionals/https-enable.test.ts

```diff
@@ -30,14 +30,13 @@ import {
   SimpleSpanProcessor,
 } from '@opentelemetry/sdk-trace-base';
 import {
-  NETTRANSPORTVALUES_IP_TCP,
-  SEMATTRS_HTTP_CLIENT_IP,
-  SEMATTRS_HTTP_FLAVOR,
-  SEMATTRS_HTTP_STATUS_CODE,
-  SEMATTRS_NET_HOST_PORT,
-  SEMATTRS_NET_PEER_PORT,
-  SEMATTRS_NET_TRANSPORT,
+  ATTR_CLIENT_ADDRESS,
+  ATTR_HTTP_RESPONSE_STATUS_CODE,
+  // ATTR_NETWORK_TRANSPORT,
+  ATTR_SERVER_PORT,
+  ATTR_NETWORK_PROTOCOL_NAME,
 } from '@opentelemetry/semantic-conventions';
+// import { NET_TRANSPORT_VALUE_IP_TCP } from '@opentelemetry/semantic-conventions/incubating';
```

Comment:
Reminder to remove some of these commented imports in the tests.

Reply:
Will do. Just waiting to hear back from @trentm on some of my questions about `NET_TRANSPORT_VALUE_IP_TCP` before I remove it.